atomic_reactor/utils/koji.py

```diff
@@ -581,11 +582,11 @@ def get_repositories_and_digests(workflow, pullspec_image):
     """
     digests = {}  # image -> digests
     typed_digests = {}  # media_type -> digests
-    for registry in workflow.push_conf.docker_registries:
+    for registry, registry_conf in workflow.conf.registries.items():
```

Comment:
Same here as above: would `for _, registry_conf in workflow.conf.registries.items():` also work?

Reply:
updated